presto-docs/src/main/sphinx/release/release-0.258.rst

```diff
@@ -0,0 +1,44 @@
+=============
+Release 0.258
+=============
+
+**Highlights**
+==============
+
+**Details**
+===========
+
+General Changes
+_______________
+* Fix issues in SQL functions where query might fail with compiler error when function implementation includes lambda variable that has the same name as column names or function input names.
+* Fixes a bug in Parquet dereference pushdown. The issue causes column name not picked up properly, resulting in inconsistent query result in certain cases.
```

Comment:
This should be in Hive changes

`Fix a bug in Parquet dereference pushdown that caused inconsistent query results in certain cases`